### ISSUE TITLE
Restart with FDS 6.7.8

### ISSUE THREAD

**Issue by rupole1185** [firemodels/fds#10656]
Hi there,

as you may be expecing, I am having a very close look at the RESTART feature of FDS. On the new FDS 6.7.8 I found a new issue with the restart file. 

This time the simulation run for 230s and then a restart occured and so it restarted from 200s. The simulation run with fds_openmp, 4 cores and 2 threads

After the restart the results are a bit messed up:

* Fire after restart seems to be moving inside of the domain

![image](https://user-images.githubusercontent.com/7260681/172334597-61488d8a-8b2a-45a3-a404-19e3996d9c68.png)

* Smoke is "flasing" [like it disappears from certain cells to appear on others]

![image](https://user-images.githubusercontent.com/7260681/172335193-b02ff9d9-8b7f-4816-a0a6-4755786749a9.png)

* Velocities slices are confused and the time at the botton is incorrect

![image](https://user-images.githubusercontent.com/7260681/172335405-ba17bd3c-0e15-4329-855a-6f99defd0bf7.png)

I attach here the original file that generated these results. I am now working on reducing the problem size so it can be easier to reproduce. 

[UCL_-_Fire2_-_1F_-_FF_-_R02.fds.txt](https://github.com/firemodels/fds/files/8851509/UCL_-_Fire2_-_1F_-_FF_-_R02.fds.txt)

**Comment by rmcdermo** [comment]
This could be a visualization problem.  Please make sure that the lower limit for loading SMOKDE3D HRRPUV is the same in each case.

<img width="607" alt="Screen Shot 2022-06-07 at 5 48 57 AM" src="https://user-images.githubusercontent.com/4418497/172351094-2ec38491-44e1-4f40-914e-47db2369d66a.png">

**Comment by rupole1185** [comment]
Hi Randy, no it's not that the case (otherwise we wouldn't get an explanation for the fire moving around the domain). I attach here the whole set of results so you can check them yourself: https://we.tl/t-xHtlvJ7JK8

**Comment by rupole1185** [comment]
I have a good news and a bad one. I have generated a simplified file (by just reducing the mesh dimensions in terms of number of cells). With this case I run the same situation above (simulation starts, killed at about 230s and restarted) but I didn't manage to reproduce the issue.

[UCL_-_Fire2_-_1F_-_FF_-_R02.fds.txt](https://github.com/firemodels/fds/files/8853118/UCL_-_Fire2_-_1F_-_FF_-_R02.fds.txt)

I'll try again but this make me wonder: could it be that in the original run the solver has been killed while writing the output files? In fact, to be honest, it seems more a corruption of these output files rather than an incorrect simulation.

**Comment by rmcdermo** [comment]
I guess that's possible.

@drjfloyd At some point did we not have a way to retain more than one set of restart files?  I'm looking the guide and code and don't see it.  But seems we had a way to have a backup in the case the present files got corrupted.

**Comment by rupole1185** [comment]
Let me just say that the restart seemed to work ok. It's the smokeview that got corrupted file (i.e. slices, smoke, HRR, ..)

**Comment by drjfloyd** [comment]
I don't think we ever had an option to preserve more than one set of restart files.

**Comment by rmcdermo** [comment]
Glenn is out this week.  Leave this open and maybe he will have some idea how to make SMV restart more robust.

**Comment by mcgratta** [comment]
Someone requested a multiple restart option but I thought that we have enough problems already with restarts. Glenn and I have worked on various improvements to the outputting of data, so unless there is something obvious, I don't know if we can fix whatever it was that first prompted the issue.

**Comment by gforney** [comment]
Smokeview doesn't do anything special to deal with an fds run that has restarts.  Suppose in case 1, fds runs from 0 to 100 s without any restarts. In case 2 fds runs the same case from 0 to 50 s then stops generating a restart, then continues on from 50 to 100 s. Are the files smokeview sees any different in these two cases? ( Slice 3d smoke etc)

**Comment by mcgratta** [comment]
That's usually not how things go. FDS runs for 75 s and then there's a power outage. The last restart file saved was at 50 s. Does Smokeview delete the old results from 50 to 75 s? For the .csv files, FDS does go back and delete the data.

**Comment by gforney** [comment]
Smokeview does not delete any data. Ill have to look more closely at what
smokeview does with cases that are restarted when i get back.

**Comment by rupole1185** [comment]
Hi there! I have one question: could this issue be linked to this one https://github.com/firemodels/fds/issues/10677 . I have just seen that FDS 6.7.9 solved it and I wonder it could be the solution for mine as well.

**Comment by mcgratta** [comment]
Issue #10677 involved a case where the FDS simulation finished normally, but the user wanted to restart the case from an earlier time. This did not work because FDS was dumping a restart file at the end of the run. This problem was fixed in 6.7.9.

**Comment by Er9y714** [comment]
A behaviour I noticed with SMV, maybe it is related to this issue. 

- Calculation starts running
- I open SMV and load smoke
- calculation continues 
- I update the results (u key) (not unloading and loading smoke again)
- smoke is only shown in the meshes that were loaded in the first try. Even the smoke propagates all domain, I only see those initially loaded meshes. 

@rupole1185 , The reason I shared this is to ask if you are running SMV or loading the smoke **after** the calculation is finished.

Moving flame is often an indication of underventilated fire in FDS.

**Comment by drjfloyd** [comment]
Issue has gone stale. If this is still an issue with the current release it can be opened as a new issue with the current symptoms.
